### PR TITLE
feat(_dbs_info): Added _dbs_info method

### PR DESCRIPTION
**Pull request by jpinkster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,60 @@ couchDb.getDatabases()
 ]
 ```
 
+### findDatabaseInfo() [POST \_dbs\_info](http://docs.couchdb.org/en/latest/api/server/common.html#_dbs_info)
```
Comment (jpinkster) [inline README.md]: This link to their docs is not right .. where did you get it from or did you make it up? It should be 
http://docs.couchdb.org/en/latest/api/server/common.html#dbs-info

File: README.md [hunk 2]
```diff
@@ -105,6 +105,60 @@ couchDb.getDatabases()
...
+This endpoint gets information about the specified databases.
```
Comment (jpinkster) [inline README.md]: You should just drop in the info that couchdb provides 
Returns information of a list of the specified databases in the CouchDB instance. This enables you to request information about multiple databases in a single request, in place of multiple GET /{db} requests.

File: README.md [hunk 3]
```diff
@@ -105,6 +105,60 @@ couchDb.getDatabases()
...
+
+**Request Example:**
+```javascript
+couchDb.getDatabases({keys:['key1','key2']})
```
Comment (jpinkster) [inline README.md]: Your example needs to be updated

File: src/couchdb.ts [hunk 4]
```diff
@@ -1,8 +1,8 @@
-import * as request from 'request-promise';
+import { CouchDbOptions, CouchDbResponse } from './index';
```
Comment (jpinkster) [inline src/couchdb.ts]: Lets check your auto import order .. this is auto moving things differently than tyler and i

File: src/couchdb.ts [hunk 5]
```diff
@@ -113,6 +113,31 @@ export class CouchDb {
...
 
+    /**
+     * Get the list of specified databases info.
+     * @param {{
+     *         keys: string[];
+     *     }} options
+     * @return {Promise}
+     */
+    findDatabaseInfo(options: {
+        keys: string[]
```
Comment (jpinkster) [inline src/couchdb.ts]: Lets not call this `keys` .. people wont really know what that means but looking at their docs, it appears to be db names so we should call this property `dbNames`

**Comment by jpinkster** [comment]
@Bern222 Not sure if you were ready for this to be a PR but i saw the open branch and didnt want it just sitting there..

Here is some feedback. Other comments:
- Keep branch names to dashes
- Our current server is at 2.1.1 and this feature is for 2.2.0+ .. we will need to upgrade in order to test this

**Comment by Bern222** [comment]
This wasn’t finished.  It wasn’t tested yet because I moved back to loop on Tuesday.  I can make these changes and test it.

File: src/couchdb.ts (hunk 5 above)
Comment (Bern222) [reply src/couchdb.ts]: Doesn't the endpoint need this to be names keys?  

Request JSON Object:
keys (array) – Array of database names to be requested

Comment (jpinkster) [reply src/couchdb.ts]: Yes but that doesnt mean our function has to match their call signature

**Comment by jpinkster** [comment]
moving away from this for now ...

**Pull request closed without merging**